tox.ini

```diff
@@ -75,7 +75,6 @@ commands =
   black .
 
 [testenv:pep8]
-commands_pre =
```

Comment:
?

Reply:
Read the commit title; we need the deps to be installed to have the definition of the libraries we depends on. :)